Terminal/Terminal.sol

```diff
@@ -0,0 +1,60 @@
+pragma solidity >= 0.6.0;
+
+interface ITerminal {
+	
+	function inputStr(uint32 answerId, string prompt, bool multiline) external returns (string value);
+	function inputInt (uint32 answerId, string prompt) external returns (int256 value);
```

Comment:
Please, remove inputInt and create an independent `NumberInput` 
@BigRoma, please create a proposal.